lib/kino/control.ex

```diff
@@ -34,6 +34,8 @@ defmodule Kino.Control do
 
   @type t :: %__MODULE__{attrs: Kino.Output.control_attrs()}
 
+  @type interval :: {:interval, milliseconds :: non_neg_integer()}
```

Comment:
```suggestion
  @opaque interval :: {:interval, milliseconds :: non_neg_integer()}
```

Reply:
I recently thought we should do that for all widget types, since the internal format shouldn't be relevant to the user, wdyt?

Reply:
Sounds good to me!